pandas/tests/sparse/test_format.py

```diff
@@ -3,10 +3,11 @@
 
 import numpy as np
 
+from pandas._config.config import option_context
```

Comment:
would prefer you import from pandas directly (and all cases where it is possible)